coalib/bearlib/languages/Language.py

```diff
@@ -290,15 +311,18 @@ class Language(metaclass=LanguageMeta):
     >>> @Language
     ... class TrumpScriptDerivative(Language.TrumpScript):
     ...     __qualname__ = 'Shorter'
-    ...     comment_delimiter = '//'
+    ...     comment_delimiters = '//',
     ...     keywords = None
 
     >>> Language.TrumpScriptDerivative()
     Shorter 2.7, 3.3, 3.4, 3.5, 3.6
 
     >>> Language.TrumpScriptDerivative().get_default_version().attributes
-    ['comment_delimiter', 'keywords', 'string_delimiter']
+    ['comment_delimiters', 'keywords', 'string_delimiter']
     >>> Language.TrumpScriptDerivative().get_default_version().keywords
+    >>> Language.TrumpScriptDerivative().get_default_version() \
+.comment_delimiters
```

Comment:
this layout isnt acceptable.

worse case, you can save `Language.TrumpScriptDerivative().get_default_version()` to a variable in the doctest and then use that to shorten this line.